File: Common/Model/SearchResultItemModel.swift
//
//  SearchResultItemModel.swift
//  ReactiveSwiftSample
//
//  Created by Masanori Nakano on 2021/12/20.
//

import Foundation

final class SearchResultItemModel: NSObject {
    let title: String
    let pageId: PageId
    
    // MARK: - NSObject

    init(item: SearchResponse.Query.Item) {
        self.title = item.title
        self.pageId = item.pageid

        super.init()
    }
}

File: AACSample/app/src/main/java/work/d128/aacsample/model/SearchResultItemModel.kt
package work.d128.aacsample.model

import work.d128.aacsample.entity.search_response.Item

class SearchResultItemModel(item: Item) {
    val title: String
    val pageId: Int

    init {
        this.title = item.title
        this.pageId = item.pageid
    }
}